ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -1125,6 +1127,29 @@ private void endCompaction() throws StoreException {
     if (compactionLog != null) {
       if (compactionLog.getCompactionPhase().equals(CompactionLog.Phase.DONE)) {
         logger.info("Compaction of {} finished", storeId);
+        if (srcIndex != null && !srcIndex.isEmpty() && !compactionLog.cycleLogs.isEmpty()) {
+          // The log segment positions after compaction
+          Set<Long> compactedLogSegments = srcIndex.getLogSegments()
```

Comment:
Still think the size of `compactedLogSegments` is good enough to give us a good reference. 

Also, suggest rename it to `logSegmentsAfterCompaction`